pkg/util/version.go

```diff
@@ -25,6 +37,17 @@ var (
 	AppVersion   = "canary"
 )
 
+func VersionInfo() Version {
+	return Version{
+		GitVersion:   AppVersion,
+		GitCommit:    AppGitCommit,
+		GitTreeState: AppGitState,
+		GoVersion:    runtime.Version(),
+		Compiler:     runtime.Compiler,
+		Platform:     fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
+	}
+}
+
 func version() string {
```

Comment:
May be one for a follow up, but can we remove this function now? `canary` should never be set as the version tag anymore since the build infra changes. May be easiest to follow up though 😄